### DISCUSSION TITLE
Renovatebot doesn't bump the version number in actions arguments

### DISCUSSION THREAD

**Discussion by Eric-Jalal** [renovatebot/renovate#24347] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Renovatebot doesn't bump up the version numbers that are passed as an argument for example this:
```json
      - uses: actions/setup-node@v3
        with:
          node-version: "20"
```
Is there any configuration that we are missing or it is not yet supported?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
What are you expecting it to be bumped to?

**Reply by HonkingGoose** [reply]
There's a new `actions/setup-node@v4` available, so maybe Renovate is not seeing that somehow... this is unlikely, because I got the update from Renovate already on a personal repo.

Or they're asking about the `with: node-version: "20"` part. I remember we don't support that `with:` field in GitHub Actions (yet), we already have an issue/discussion that mentions the `with:` field, but I can't find it now...

Relevant docs: https://docs.renovatebot.com/node/
